### PR TITLE
1.17.1 util.math stuff

### PR DESCRIPTION
**Pull request by apple502j**
There are also some methods that I vaguely know what it is meant to be, but cannot come up with a good name...

### CODE REVIEW

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 1]
```diff
@@ -6,6 +6,13 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 	FIELD field_15726 RANDOM Ljava/util/Random;
 	FIELD field_15727 ARCSINE_TABLE [D
 	FIELD field_15728 SMALLEST_FRACTION_FREE_DOUBLE D
+	FIELD field_29844 PI F
+	FIELD field_29845 HALF_PI F
+	FIELD field_29846 TAU F
```
Comment (enbrain) [inline mappings/net/minecraft/util/math/MathHelper.mapping:11]: I'd document that it's `2 * PI`.

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/util/math/ChunkPos.mapping [hunk 2]
```diff
@@ -35,6 +35,8 @@ CLASS net/minecraft/class_1923 net/minecraft/util/math/ChunkPos
 		ARG 1 offsetX
 		ARG 2 y
 		ARG 3 offsetZ
+	METHOD method_37232 toLong (Lnet/minecraft/class_2338;)J
```
Comment (liach) [inline mappings/net/minecraft/util/math/ChunkPos.mapping:38]: Hmm, how is this different from the instance method toLong on block pos

Comment (liach) [reply mappings/net/minecraft/util/math/ChunkPos.mapping:38]: Hmm, this one converts a block pos to a chunk long, guess this name is fine

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 3]
```diff
@@ -88,6 +88,8 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 		ARG 0 matrix
 		ARG 1 direction
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD method_30928 pointsTo (F)Z
+		ARG 1 degrees
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping]:
```suggestion
		ARG 1 yaw
```

File: mappings/net/minecraft/util/math/Direction.mapping [hunk 4]
```diff
@@ -88,6 +88,8 @@ CLASS net/minecraft/class_2350 net/minecraft/util/math/Direction
 		ARG 0 matrix
 		ARG 1 direction
 	METHOD method_23955 getUnitVector ()Lnet/minecraft/class_1160;
+	METHOD method_30928 pointsTo (F)Z
```
Comment (liach) [inline mappings/net/minecraft/util/math/Direction.mapping:91]: This rather returns if the current direction forms an acute angle with the given yaw value. Better note that this always return `false` for vertical directions.

**Review by YanisBft — APPROVED** [review]

File: mappings/net/minecraft/util/math/MathHelper.mapping [hunk 5]
```diff
@@ -9,8 +9,10 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 	FIELD field_29844 PI F
 	FIELD field_29845 HALF_PI F
 	FIELD field_29846 TAU F
+		COMMENT Tau is equal to {@code 2 * PI}.
 	FIELD field_29847 RADIANS_PER_DEGREE F
 	FIELD field_29848 DEGREES_PER_RADIAN F
+	FIELD field_29849 APPROXIMATE_EQUALITY_TOLERANCE F
```
Comment (liach) [inline mappings/net/minecraft/util/math/MathHelper.mapping]: We usually call this 
```suggestion
	FIELD field_29849 EPSILON F
```
Look up &epsilon; in math.

**Review by liach — APPROVED** [review]

**Review by enbrain — APPROVED** [review]

**Comment by Juuxel** [comment]
`canMakeAcuteAngle` sounds really awkward. Really, while the acute angle property of the method is true, a more user-friendly interpretation of it is whether the yaw rotation roughly faces this direction.

**Comment by apple502j** [comment]
Hmm, cannot come up with a good name.

**Comment by Juuxel** [comment]
IMO `pointsTo` was good already.

**Review by liach — APPROVED** [review]

**Pull request merged**